src/raylib.h

```diff
@@ -1226,6 +1226,8 @@ RLAPI Rectangle GetCollisionRec(Rectangle rec1, Rectangle rec2);
 // NOTE: This functions do not require GPU access
 RLAPI Image LoadImage(const char *fileName);                                                             // Load image from file into CPU memory (RAM)
 RLAPI Image LoadImageRaw(const char *fileName, int width, int height, int format, int headerSize);       // Load image from RAW file data
+RLAPI Image LoadImageSvg(const char* fileName);                                                          // Load image from SVG file data with default size from SVG
```

Comment:
I prefer to keep base SVG loading inside `LoadImage()` as an additional supported file format.

Reply:
I've moved loading partially to inside LoadImage(). This will load the .svg file now at the defined size inside the .svg file.